src/manifests/svc.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# Source: https://github.com/ory/k8s/k8s/helm/charts/kratos/templates/service-admin.yaml
+# helm charm v0.25.3
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ app_name }}-admin
+  labels:
+    app.kubernetes.io/component: {{ app_name }}-admin
+    app.kubernetes.io/name: {{ app_name }}
+    app.kubernetes.io/managed-by: juju
+    app.juju.is/created-by: {{ app_name }}
```

Comment:
Is the presence of these labels supposed to help in resources removal by juju? They don't get cleaned up when the charm is removed. Maybe we need controller or model info in the metadata too? I will check this as I have same pb in hydra.

Reply:
```yaml
operator.juju.is/name=<app_name>   
operator.juju.is/target=application
```
These two are the important ones for removal. It only work on namespaced resources. 
These are covered (I think)
- Service
- Secret
- role binding, role
- custom resource / definition
- config map

Reply:
Those services don't get deleted on remove event, but I guess that's a juju bug

Reply:
I added an on remove handler to remove them. Good catch